plutip-server/Types.hs

```diff
@@ -95,6 +102,8 @@ instance FromJSON Lovelace where
 data StartClusterRequest = StartClusterRequest
   { slotLength :: NominalDiffTime
   , epochSize :: EpochSize
+  , maxTxSize :: Natural
```

Comment:
I'm against making more of these mandatory. The users may want to start clusters manually, in this case coming up with all the parameters might be too much of a blocker. Can we make all parameters that are not essential (all except keysToGenerate) optional?

Reply:
fair point. I agree. See [8c11d0e](https://github.com/mlabs-haskell/plutip/pull/165/commits/8c11d0e88ee19b6010fb9a17592ba3b4809c8794)